routes/web.php

```diff
@@ -60,21 +60,20 @@
 /*
 *   Controller Dashboard
 */
-Route::prefix('dashboard')->middleware('auth')->group(function() {
+Route::prefix('dashboard')->middleware('auth')->group(function () {
     Route::get('/', 'ControllerDash@dash');
 
-    Route::prefix('controllers')->group(function() {
-      
+    Route::prefix('controllers')->group(function () {
         Route::get('/mentoravi', 'TrainingController@showMentAvail');
         Route::post('/mentoravi', 'TrainingController@saveSession');
         Route::get('/trainingreq', 'TrainingController@showRequests');
         Route::delete('/trainingreq/{id}/delete', 'TrainingController@cancelSession');
         Route::get('/sessions_today', 'TrainingController@showsessions_today');
 
```

Comment:
While we're cleaning up formatting, will you please remove all of these extra lines in the routes file? I'm not sure why these are in here. It's not your fault, but might as well take this opportunity to clean it up.

Reply:
Resolved in c519387.